osu.Game/Skinning/Components/BeatmapInfoDrawable.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Please remove these and adjust classes. We don't add NRT disable to new classes.